File: examples/py-fixtures/crashes.py
"""Dies on an uncaught exception. Python's answer to c-fixtures/crashes.c.

There is no segfault to reach from Python, and an uncaught exception is the
same thing at this level: the program stops because of something it did, not
because it was asked to, and the process exits non-zero.
"""

print("about to fail", flush=True)
raise ValueError("nothing here")  # line 9 — the failure

File: examples/go-fixtures/crashes.go
// Dies on an unrecovered panic. Go's answer to c-fixtures/crashes.c.
//
// There is no segfault to reach from safe Go, and an unrecovered panic is the
// same thing at this level: the program stops because of something it did, not
// because it was asked to, and the process exits non-zero (Go uses 2).

package main

import "fmt"

func main() {
	fmt.Println("about to fail")
	panic("nothing here")
}
